### PR TITLE
Feature/issue 1023 add aria roles

### PR DESCRIPTION
**Pull request by SteveBarnett**
Ready for review. @justinvdm, got a minute?

### CODE REVIEW

**Comment by SteveBarnett** [comment]
Tests updated. @justinvdm, can you review, please?

File: go/tests/test_views.py [hunk 1]
```diff
@@ -75,9 +75,19 @@ def test_help_not_authenticated(self):
...
         self.assertContains(response, 'analytics.js')
+
+    def test_aria_footer(self):
+        response = self.client.get(reverse('home'), follow=True)
+        self.assertContains(response, 'role="contentinfo"')
+
+    def test_aria_banner_and_nav(self):
+        response = self.client.get(reverse('home'), follow=True)
+        self.assertContains(response, 'role="banner"')
+        self.assertContains(response, 'role="navigation"')
```
Comment (justinvdm) [inline go/tests/test_views.py:95]: A bit of a bikeshed, but maybe these should be separate tests since they are testing separate things?

**Comment by justinvdm** [comment]
Looks good, small comment about tests, :+1: otherwise.

File: go/tests/test_views.py (hunk 1 above)
Comment (SteveBarnett) [inline go/tests/test_views.py:95]: After discussion with @justinvdm IRL, we're going to leave these for now, but remove once #1029 is done.

**Comment by justinvdm** [comment]
:+1:!

**Pull request merged**